ui/src/plot.tsx

```diff
@@ -1189,13 +1193,16 @@ interface State {
   animate?: B
 }
 
-export const View = bond(({ name, state, changed }: Model<State>) => {
+export const View = bond(({ name, state, changed }: Model<State & { commands?: Command[] }>) => {
   const render = () => {
-    const { title = 'Untitled', plot, data, events, interactions, animate } = state
+    const { title = 'Untitled', plot, data, events, interactions, animate, commands } = state
     return (
       <div className={css.card}>
-        <div className='wave-s12 wave-w6'>{title}</div>
-        <div className={css.body}>
+        {title && <div className='wave-s12 wave-w6'>{title}</div>}
+        <div
+          className={css.body}
+          style={{ paddingTop: title ? TITLE_PADDING_TOP : commands?.length ? TITLE_PADDING_TOP + TITLE_HEIGHT : 0 }}
```

Comment:
Can we move the spacing into title instead of body? This way, we should be able to get rid of the spaghetti condition.

Also, spacing should be the same regardless of commands presence. Just make sure that it's enough so that commands do not overlap. You can also think about moving commands slightly more into corner instead of only adjusting the title spacing.

Reply:
> Can we move the spacing into title instead of body? This way, we should be able to get rid of the spaghetti condition.

But this won't cover the case where commands are present but title is not because if title is not rendered, the spacing is not taking effect either. Or am I missing something?

Reply:
> But this won't cover the case where commands are present but title is not because if title is not rendered, the spacing is not taking effect either.

Correct. Ignore my prev comment then.